test/e2e/policies.test-suite/topology-aware/nri-resmgr.cfg

```diff
@@ -0,0 +1,8 @@
+policy:
+  Active: topology-aware
+  ReservedResources:
+    CPU: 750m
+logger:
+  Debug: nri-resmgr,resource-manager,cache,policy
```

Comment:
why are we adding this policy here? Is it going to be used as a default one?

Reply:
Do you mean why `policy.Active` is specified ?

Actually specifying `policy.Active` is not required any more (ever since commit aae48894a3a53faa96d0a53a3b7aa0cdd3d4ff9f), since we are now going with a one policy per binary model. If you specify the policy, it will just ensure that you are indeed running with the expected binary and the right policy. The binary will refuse to start up if there is a given policy name that does not match the one the binary has been built with.

Reply:
No, sorry my point was about why are we adding this configuration file itself in here? 
But thanks for the previews comment @klihub , because I learned something new 😁

Reply:
The e2e tests will use that file as a default one if the test does not specify anything else. The tests were flaky before because by default we were using in the individual tests, the cfg file from previous test, which then caused issues.
So I added the default nri-resmgr.cfg file so that each test will start with this file (unless the test specifies its own cfg file) and will get a fresh start.